src/components/report/TextControls.css

```diff
@@ -26,63 +26,44 @@
   font-size: 0.85rem;
 }
 
-/* TODO: componentalizationreuse */
-.button {
-  padding: 0.25rem 0.5rem;
-  margin: 0;
-  border: 0;
-  background: none;
-  font-size: 0.85rem;
-  background: var(--unlighter-grey);
-  color: var(--dark-grey);
-  cursor: pointer;
-  transition: background 100ms;
-  user-select: none;
-  min-width: 2em;
-  background-repeat: no-repeat;
-  background-position: center;
-  background-size: 65%;
+
+.alignment-buttons {
+  display: flex;
+  align-items: flex-start;
 }
 
-.button:hover {
-  background-color: var(--unlighterer-grey);
+.alignment-buttons .button {
+  border-radius: 0;
+  min-width: 0;
 }
 
-.active {
-  background-color: var(--green);
+.button.left,
+.button.center,
+.button.right,
```

Comment:
Could you go with „start” and „end” instead of left/right?
That will help with internationalisation / left-to-right writing systems (in case you ever walk down that route).
I adopted it personally as a Good Practice.

Reply:
These are explicitly for left and right, not start and end, regardless of writing direction.